OuterWildsRandomSpeedrun/SpawnPointPool.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace OuterWildsRandomSpeedrun
+{
+    public struct SpawnPointConfig
+    {
+        public string internalId;
+        public string displayName;
+    }
+
+    public class SpawnPointPool
+    {
+        public List<SpawnPointConfig> SpawnPointConfigs
```

Comment:
Change return type to the interface here: `IList<SpawnPointConfig>`

Reply:
Done in bd51e7d